README.md

```diff
@@ -473,6 +473,8 @@ do {
 
 * **3.1.15** If you have a function that takes no arguments, has no side effects, and returns some object or value, prefer using a computed property instead.
 
+* **3.1.16** For the purpose of namespacing a set of `static` functions and/or computed properties, prefer using a caseless `enum` over a `class` or a `struct`. This way, you don't have to add a `private init() { }` to the container.
```

Comment:
maybe repeat the word `static` again. It seems not quite clear whether `static` refers to "functions" only or not.